README.md

```diff
@@ -1,2 +1,51 @@
 # movie-store-app
 This is the repository for a movie store app
+
+
+## Home Route:
+> / or /home for main page.
+
+## Movie Search Routes:
```

Comment:
It's definitely a good idea to get familiar with Swagger as it can help documenting your API as well as allow support for calling your endpoints without the need of using postman for instance. It's a cool tool to try in the future.